### PR TITLE
Add ubiquitous doodle placeholder

### PR DESCRIPTION
**Pull request by franciscosmit04-eng**
Adds the requested empty `ubiquitous-doodle.md` placeholder at the repository root.

### CODE REVIEW

**Comment by franciscosmit04-eng** [comment]
#include <nvtx3/nvToolsExtPayload.h>
#include <nvtx3/nvToolsExtPayloadHelper.h>

NVTX_DEFINE_STRUCT_WITH_SCHEMA_AND_REGISTER(domain, Step, "training.step",
    NVTX_PAYLOAD_ENTRIES(
        (uint64_t, iteration, TYPE_UINT64, "Training step index"),
        (float,    loss,      TYPE_FLOAT,  "Training loss")
    )
)

Step value = {42, 0.125f};
nvtxPayloadData_t pld = {Step_schemaId, sizeof(value), &value};
nvtxMarkPayload(domain, &pld, 1);
